Story 28: Infinite scrolling in lists

Rolo123y opened the pull request:
Changes made:
selectBoard.php 
-> added custom CSS for the cards so that they do not mix with the cards in the listView.php page

listView.php 
-> Cards have a fixed height and a min-width
-> Allows for scrolling through a list once an overflow in the list

Rolo123y committed:
Story 28: Infinite scrolling in lists

Changes made:
selectBoard.php -> added custom CSS for the cards so that they do not
                   mix with the cards in the listView.php page

listView.php -> Cards have a fixed height and a min-width
             -> Allows for scrolling through a list once an overflow in
	        the list

AaronDzf reviewed (changes requested):
Scroll feature looks good!
I checked the archive board and noticed scrolling doesn't exist on there, maybe we should look into that?

Tested the scrolling feature for the following:
• Scroll bar does not exist when number of tasks does not exceed the height of the task list card
• Scroll bar does appears when adding a task will cause the task list card to exceed height limit
• Scroll bar disappears when archiving a task will cause the task list card to be below the height limit
• Scroll bar functions correctly (moves up and down)

Rolo123y committed:
ArchivePage.php

Fixed:
-> When archiving, the task only appears for the archivepage for that
boardID only. Before, the archived task would appear for all boardID
which means all Boards would have the same archived list.
-> Card for the ArchivePage now has scrollable content

Rolo123y commented:
The archive page is now attached to a boardID. and the scrollable content is also implemented.

AaronDzf reviewed (commented):
Good changes to the archive board!
Scroll feature works correctly on this board similar to the main boards.

The query change looks like it properly retrieves archived tasks for each archive board individually now!
1) Archived task on a board only shows up in the archive board associated with that board
2) Unarchiving a task on an archive board will only show up on the board it was associated with originally
